yum/beaker-server-RedHatEnterpriseLinux.repo

```diff
@@ -3,7 +3,7 @@ name=Beaker Server - RedHatEnterpriseLinux$releasever
 baseurl=https://beaker-project.org/yum/server/RedHatEnterpriseLinux$releasever/
 enabled=1
 gpgcheck=1
-gpgkey=https://beaker-project.org/gpg/RPM-GPG-KEY-redhatengsystems https://fedoraproject.org/static/352C64E5.txt
+gpgkey=https://beaker-project.org/gpg/RPM-GPG-KEY-redhatengsystems https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
```

Comment:
Would be better to use $releasever.. For EPEL 8

Reply:
We can follow it here - https://github.com/beaker-project/beaker-project.org/issues/43
It is good for now. We still have a long way to go.